src/internal/incompatibility.rs

```diff
@@ -44,6 +44,9 @@ enum Kind<P: Package, V: Version> {
     DerivedFrom(usize, usize),
 }
 
+/// A type alias for a pair of [Package] and a corresponding [Term].
+pub type PackageTerm<P, V> = (P, Term<V>);
```

Comment:
Ok

Reply:
But I don't see much value. `(P, Term<V>)` seems pretty straightforward

Reply:
Sometimes adding a level of indirection is not necessary

Reply:
I admit this was just to fix a clippy lint, both before and after are quite easy to read IMO. I weighted the options of adding a suppress for the lint, and defining a type alias for PackageTerm and went with the latter.

The reason was we already have this meaning implicitly defined in our library: we have `package_terms` map that we iterate over, and the iterator returns this tuple. So it didn't feel like a new thing you have to look up a definition of.

If you disagree with the reasoning, let me know and I will change it to a lint suppression instead.

Reply:
Alright, I guess both are ok. Choose what you prefer :)

Reply:
Thanks. Let's keep a type alias then :)